usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -389,9 +395,13 @@ def nightly_loader(self, start_date):
 
         logger.info("FPDS NIGHTLY UPDATE COMPLETE")
 
-    def load_specific_transactions(self, detached_award_procurement_ids):
+    def load_specific_transactions(self, explicit_ids, ids_file):
```

Comment:
I recommend keeping `load_specific_transactions()` as-is. Combine the file and broker ids before invoking this method and pass the single list.